### ISSUE TITLE
Official Wii U Gamecube Adapter not working

### ISSUE THREAD

**Issue by nvitaterna** [libretro/Lakka#287]
After looking around a bit i found this https://github.com/ToadKing/wii-u-gc-adapter. Unfortunately I cannot install it as make is not available. Wondering if this can get added in. Thanks.

**Comment by Kivutar** [comment]
Do you see the gamepad appearing as /dev/js0 when you plug it?

Do you see it in the output of the lsusb command ?

Have you tried the manual configuration in RetroArch ?

**Comment by hizzlekizzle** [comment]
Unless/until a driver for the USB adapter gets added to the kernel (dunno if anyone is working on that), you'll have to run a listening daemon, which ToadKing's program is/does. If that's a thing you want to add to Lakka, it would need to be run as root.

**Comment by Kivutar** [comment]
Well, I would prefer to wait until a kernel side driver gets merged and backport the patches.

**Comment by hizzlekizzle** [comment]
I agree. Adding a separate daemon just for this one piece of hardware would be unwise, i think.
